docs/install/kata3.0-installation-guide.md

```diff
@@ -0,0 +1,63 @@
+# Kata Containers 3.0 installation guides
+The following is an overview of the different installation methods available. 
+
+## Prerequisites
+
+Kata Containers requires nested virtualization or bare metal. Check 
+[hardware requirements](/src/runtime/README.md#hardware-requirements) to see if your system is capable of running Kata 
+Containers.
+
+### Platform support
+
+Kata Containers 3.0 currently runs on 64-bit systems supporting the following
+architectures:
+
+| Architecture | Virtualization technology |
+|-|-|
+| `x86_64`| [Intel](https://www.intel.com) VT-x |
+| `aarch64` ("`arm64`")| [ARM](https://www.arm.com) Hyp |
```

Comment:
For the other platforms, could you add a [note](https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#notes) and mention the tracking issue: https://github.com/kata-containers/kata-containers/issues/4320.

Reply:
fixed